internal/xds/translator/listener.go

```diff
@@ -30,7 +37,8 @@ func buildXdsListener(httpListener *ir.HTTPListener) (*listener.Listener, error)
 		},
 		// Use only router.
 		HttpFilters: []*hcm.HttpFilter{{
-			Name: wellknown.Router,
+			Name:       wellknown.Router,
+			ConfigType: &hcm.HttpFilter_TypedConfig{TypedConfig: routerAny},
```

Comment:
@lizan @zirain is this tied to a new API change or API implementation change ? I didn't need this earlier ?

Reply:
IIRC, it's here https://www.envoyproxy.io/docs/envoy/v1.22.5/version_history/v1.22.0

![image](https://user-images.githubusercontent.com/4354057/187324360-5f1c184d-bc66-4beb-a7f9-b83a21fa4bb9.png)